src/support/mixins/color-modes.scss

```diff
@@ -24,3 +24,13 @@
     }
   }
 }
+
+@mixin color-mode-var($name, $light, $dark) {
+  @include color-mode(light) {
+    --color-#{$name}: #{$light};
+  }
+
+  @include color-mode(dark) {
+    --color-#{$name}: #{$dark};
+  }
+}
```

Comment:
/cc @simurai — this is my proposal for a dotcom "escape hatch" to ensure colors get defined for all modes.

Reply:
🤔 although we would need to modify stylelint to detect the inclusion of this mixin and make whatever variable it defines "safe"